src/main/resources/interface-specification.yml

```
@@ -389,22 +389,22 @@ paths:
         schema:
           type: string
           format: uuid
-      - name: relationshipId
+      - name: userId
         in: path
-        description: The identifier of the relationship between the security operator and the consumer
+        description: The identifier of the user between the security user and the consumer
         required: true
         schema:
           type: string
           format: uuid
     delete:
-      description: Removes an operator relationship from a Client
+      description: Removes an user from a Client
```

Same for other occurrences
```suggestion
      description: Removes a user from a Client
```